### DISCUSSION TITLE
Monekin 3 released! 🚀

### DISCUSSION THREAD

**Discussion by enrique-lozano** [enrique-lozano/Monekin#11] [Announcements]
This is our first public release. We are very pleased that this project becomes something 100% Open Source. At the moment, versions for Android and Windows are uploaded, in the future the possibility of adding more platforms will be studied.

## ✨ New features

- Completely free! No more restrictions! Unlimited accounts and budgets.
- Dates, numbers... now displays in the region format of the user by default. Previously we only support the English and the Spanish formatting.
- Support for Windows.

<hr /><em>This discussion was created from the release <a href='https://github.com/enrique-lozano/Monekin/releases/tag/v3.0.0'>v3.0.0</a>.</em>

**Comment by Nicky00l** [comment]
Hola, la app es buenísima! Me gustaría que pudieran arreglar las transacciones recurrentes ya que no deja colocarlas. Gracias!

**Reply by enrique-lozano** [reply]
Buenas tardes! Muchas gracias por escribirnos y usar la aplicación!

Actualmente se está reescribiendo la aplicación desde cero para mejorar el rendimiento entre pestañas, introducir mejoras en las transacciones recurrentes y mucho más! La nueva actualización podría estar lista con suerte en uno o dos meses.

Por favor le rogamos que para evitar la pérdida de datos, mantenga sus transacciones exportadas tanto en json como en csv, ya que esta nueva actualización será tan grande que puede que no sea compatible con los datos que hay actualmente en su dispositivo.

Muchas gracias!
